### PR TITLE
Added Python Strings .isidentifier() function

### PR DESCRIPTION
**Pull request by ATREAY**
### Description

This Pull Request is associated with Issue #3279(Python Strings .isidentifier()).

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/isidentifier/isidentifier.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+## Syntax
+
+```py
+string.isidentifier()
+```
+
+- Python `identifiers` must start with a `letter (a-z, A-Z)` or an `underscore (_)` followed by letters, digits (0-9), or underscores.
+- They cannot be a Python keyword like `if`, `else`, `while`, etc.
+
+## Example1
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/isidentifier/isidentifier.md:27]:
```suggestion
## Example
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ATREAY I had a couple of comments.

**Comment by ATREAY** [comment]
Hello @SSwiniarski, I have implemented the necessary modifications. 
I kindly request that you review them.

**Review by SSwiniarski — APPROVED** [review]
@ATREAY Looks good for a second review.

File: content/python/concepts/strings/terms/isidentifier/isidentifier.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```py
+string.isidentifier()
+```
+
+- Python `identifiers` must start with a `letter (a-z, A-Z)` or an `underscore (_)` followed by letters, digits (0-9), or underscores.
+- They cannot be a Python keyword like `if`, `else`, `while`, etc.
+
+## Example
+
+Use `.isidentifier()` to check if a string is a valid Python identifier:
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/isidentifier/isidentifier.md:29]:
```suggestion
In the code below `.isidentifier()` to check if a string is a valid Python identifier:
```

File: content/python/concepts/strings/terms/isidentifier/isidentifier.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+This example results in the following output:
+
+```shell
+True
+```
+
+## Codebyte Example
+
+Use `.isidentifier()` to check if `my_string` is a valid Python identifier:
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/isidentifier/isidentifier.md:45]:
```suggestion
The following code is runnable and uses `.isidentifier()` to check if `my_string` is a valid Python identifier:
```

File: content/python/concepts/strings/terms/isidentifier/isidentifier.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+The `.isidentifier()` string method takes in a string and returns `True` if the string is a valid Python identifier, else returns `False`.
+
+## Syntax
+
+```py
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/isidentifier/isidentifier.md:20]:
```suggestion
```pseudo
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@ATREAY I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by ATREAY** [comment]
Hello @caupolicandiaz, I have made the required changes.
I kindly request that you review them.

**Review by caupolicandiaz — APPROVED** [review]
@ATREAY Looks good

**Pull request merged**